data/role/jenkins-slave-android.yaml

```diff
@@ -0,0 +1,65 @@
+---
+psick::base::linux_classes:
+  packages: psick::packages
+  andoid: profile::android
+  nodejs: profile::nodejs  
+  ssh: ''
+
+psick::profiles::linux_classes:
```

Comment:
This looks redundat as psick::openssh is already classified in common.yaml 
https://github.com/graduway/psick-hieradata/blob/production/data/common.yaml#L42